### ISSUE TITLE
Exception AbstractContentViewHelper.php – undefined array key 'tt_content:xx'

### ISSUE THREAD

**Issue by peter-neumann-dev** [FluidTYPO3/vhs#1779]
Hello everyone 👋🏻
First: thanks for all investigations in updating VHS to work with TYPO3 v11 ✌🏻🎉

With `<v:content.render column="XX" pageUid="XX" />` I get this error in frontend:

```
PHP Warning: Undefined array key "tt_content:xx" in 
/var/www/html/public/typo3conf/ext/vhs/Classes/ViewHelpers/Content/AbstractContentViewHelper.php line 222
```

I think it's the same issue as in #1777 
I'm not sure because Pull Request is closed, but is it merged?

TYPO3 v11 
php 8.0 
vhs 6.1.1

**Comment by GuidoJansenPI** [comment]
+1 #1777 fixed it for me, but is not merged vhs 6.1.2

**Comment by NamelessCoder** [comment]
Fixed via https://github.com/FluidTYPO3/vhs/pull/1777
